### PR TITLE
Move or copy files 0.0.2

### PR DESCRIPTION
**Pull request by Dave005**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by viktorbenei** [comment]
Hey @Dave005 👋 

How did you create this PR? Asking because v0.0.1 of the step shouldn't be included if you just created a new PR.

In any case, can you please remove that? Only 0.0.2 should remain

File: steps/move-or-copy-files/0.0.2/step.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+title: Move or Copy Files
+summary: |
+  Moves or copies files
+description: |
+  This Step copies or moves a single file to the desired target location. It can also be used to rename files.
+website: https://www.datenstrom.online
```
Comment (viktorbenei) [inline steps/move-or-copy-files/0.0.2/step.yml]: please use a step related website URL, where the users can learn more about the step

File: steps/move-or-copy-files/0.0.2/step.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      Path to the target file
+    is_required: true
+    summary: ""
+    title: Target File
+  output_file: null
+- move_action: copy
+  opts:
+    description: |
+      Choose between a 'move' or 'copy' action -- standard is copy
+    is_required: false
```
Comment (viktorbenei) [inline steps/move-or-copy-files/0.0.2/step.yml]: should be true if required - I guess one of these is required ;)

File: steps/move-or-copy-files/0.0.2/step.yml (hunk 1 above)
Comment (viktorbenei) [inline steps/move-or-copy-files/0.0.2/step.yml]: can be the github url of the step of course, but should be related to the step

**Comment by Dave005** [comment]
Actually i was wondering the same, i will look into that :D

**Comment by viktorbenei** [comment]
Thanks @Dave005 and let us know if we can help with anything! ;)

**Comment by viktorbenei** [comment]
Or, accept my PRs in your step and create 0.0.3 😂

**Comment by Dave005** [comment]
:D i will do that, but probably in the morning, need to get some rest :D

**Comment by viktorbenei** [comment]
Yeah, I know that feeling :P

**Comment by viktorbenei** [comment]
Closing it for now, but feel free to create a new one with the fixes @Dave005 and we'll review it ASAP! :)

**Pull request closed without merging**